p8e-api/src/main/kotlin/io/provenance/engine/batch/MailboxReaper.kt

```diff
@@ -254,7 +254,7 @@ class MailboxReaper(
     /**
      * Handles polling mailbox for new fragment requests and process accordingly.
      */
-    @Scheduled(initialDelayString = "\${reaper.inbox.delay}", fixedDelayString = "\${reaper.inbox.interval}")
+//    @Scheduled(initialDelayString = "\${reaper.inbox.delay}", fixedDelayString = "\${reaper.inbox.interval}")
```

Comment:
did you want this commented out still?

Reply:
Yeah, this branch is to get single-party contracts up and running with the new chain/new open source object-store for Vega. Commented this out to prevent mailbox from spewing errors as the public-chain branch of p8e-docker-compose doesn't have mailbox anymore.